test/queryRunner.test.ts

```diff
@@ -139,14 +174,34 @@ suite('Query Runner tests', () => {
             testQueryNotificationHandler.object,
             testVscodeWrapper.object
         );
+        queryRunner.batchSets[0] = {
+            executionElapsed: null,         // tslint:disable-line:no-null-keyword
```

Comment:
any reason why you are using null here (and other places in this PR) rather than undefined? We shouldn't make a habit of ignoring tslint rules (the exception being in the angular ignoring unused fields).